### DISCUSSION TITLE
Change order of form and table methods in Resources

### DISCUSSION THREAD

**Discussion by Rattone** [filamentphp/filament#13039] [Ideas]
I'm not the only one who thinks the "chronological" order should be table > form, as index comes before edit in a CRUD controller?
Every time I want to modify a table I make a mistake and look at the beginning of the Resource file because my brain thinks like that. Is there a reason why form comes before table?

**Comment by dissto** [comment]
You can publish the stubs and change the order to whatever you like. This is currently not documented so i have added a PR. 

You just need to run 

```bash
php artisan vendor:publish --tag=filament-stubs
```

then you can do whatever you want 😋

**Reply by Rattone** [reply]
I know I can publish the stubs and customize them.

I ask this question because I would like to suggest changing the order in Filament (and not having to do it myself for each project) but first I would like to understand if there were any reasons to arrive at the current situation and what other people think!
